### PR TITLE
RSeQC: check for division by zero

### PR DESCRIPTION
**Pull request by vladsavelyev**
Fixes https://github.com/ewels/MultiQC/issues/2033

### CODE REVIEW

File: multiqc/modules/rseqc/inner_distance.py [hunk 1]
```diff
@@ -48,6 +48,8 @@ def parse_reports(self):
         for s_name in self.inner_distance:
             self.inner_distance_pct[s_name] = OrderedDict()
             total = sum(self.inner_distance[s_name].values())
+            if total == 0:
+                continue
```
Comment (ewels) [inline multiqc/modules/rseqc/inner_distance.py:52]: Should add a warning log message here

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,7 @@
   - Do not fail when all input samples have no coverage ([#2005](https://github.com/ewels/MultiQC/pull/2005)).
 - **RSeQC**
   - Fix "max() arg is an empty sequence" error ([#1985](https://github.com/ewels/MultiQC/issues/1985))
+  - Fix division by zero on all-zero input ([#2040](https://github.com/ewels/MultiQC/pull/2040))
```
Comment (ewels) [inline CHANGELOG.md:33]:
```suggestion
  - Fix division by zero on all-zero input to inner-distance ([#2040](https://github.com/ewels/MultiQC/pull/2040))
```

**Review by ewels — APPROVED** [review]
Log message would be good, then LGTM to merge 👍🏻

**Comment by vladsavelyev** [comment]
Re log message - I don't think it's needed as the sample data will stay in the report. We just make it skip converting 0 "things" into 0 percents.

**Pull request merged**